R/bayesianStateSpace.R

```diff
@@ -339,13 +335,19 @@ quantInv <- function(distr, value){
   bstsTable$addColumnInfo(name="R2",      title =gettextf("R%s", "\u00B2"),           type= "number")
   bstsTable$addColumnInfo(name="relGof",  title=gettext("Harvey's goodness of fit"),  type= "number")
 
-  if (bstsResults$niter < options$samples)
-    bstsTable$addFootnote(message=gettextf("Test: Only %1$s draws were sampled out of the desired %2$s. Additionally, %3$s MCMC draws out of %1$s are discarded as burn in.", bstsResults$niter, options$samples, options$burn))
-  else
-    bstsTable$addFootnote(message=paste0(options$burn, " MCMC draws out of ", bstsResults$niter, " are discarded as burn in."))
+  if(!is.null(bstsResults)){
+    if (bstsResults$niter < options$samples )
+      message <- gettextf("Only %1$s draws were sampled out of the desired %2$s. Additionally, %3$s MCMC draws out of %1$s are discarded as burn in.", bstsResults$niter, options$samples, options$burn)
+    else
+      message <- gettextf(paste0(options$burn, " MCMC draws out of ", bstsResults$niter, " are discarded as burn in."))
```

Comment:
```suggestion
      message <- gettextf("%1$d MCMC draws out of %2$d are discarded as burn in.", options$burn, bstsResults$niter)
```
otherwise this cannot be translated.